File: Lib/Telerik/Examples/QuickStart/FileDialogs/FirstLook/StorageFile.cs
using System;

namespace Telerik.Examples.WinControls.FileDialogs.FirstLook
{
    public class StorageFile
    {
        public string Name { get; set; }

        public string FileSystemInfoType { get; set; }

        public DateTime UploadedTime { get; set; }

        public StorageFile(string name)
        {
            this.Name = name;
            this.FileSystemInfoType = "File";
            this.UploadedTime = DateTime.Now;
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/FileDialogs/FirstLook/StorageFile.vb
Namespace Telerik.Examples.WinControls.FileDialogs.FirstLook
    Public Class StorageFile
        Public Property Name As String
        Public Property FileSystemInfoType As String
        Public Property UploadedTime As DateTime

        Public Sub New(ByVal name As String)
            Me.Name = name
            Me.FileSystemInfoType = "File"
            Me.UploadedTime = DateTime.Now
        End Sub
    End Class
End Namespace
